nixos/modules/hardware/rasdaemon.nix

```
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  options = {
+    hardware.rasdaemon = {
+      enable = mkOption {
```

```suggestion
      enable = mkEnableOption "RAS logging daemon"
```

Get's rid of all of the boilerplate below :)